programs/clearing_house/src/controller/lp.rs

```diff
@@ -87,6 +87,43 @@ pub fn settle_lp_position(
         .checked_div(n_shares_i128)
         .ok_or_else(math_error!())?;
 
+    // account for rounding
+    let remainder_baa = remainder_base_asset_amount_per_lp
+        .checked_mul(n_shares_i128)
+        .ok_or_else(math_error!())?
+        .checked_div(AMM_RESERVE_PRECISION_I128)
+        .ok_or_else(math_error!())?;
```

Comment:
you're doing the `lp_metrics.remainder_base_asset_amount * AMM_RESERVE_PRECISION_I128 / n_shares_i128 * n_shares_i128 / AMM_RESERVE_PRECISION_I128` here again